tensorflow/lite/kernels/activations.cc

```diff
@@ -347,6 +347,26 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
   }
 }
 
+namespace {
+template <typename T>
+void QuantizedRelu1(const TfLiteTensor* input, TfLiteTensor* output) {
+  ActivationParams params;
+  int32 kMin = -1;
+  int32 kMax = 1;
+  params.activation_type = FusedActivationFunctionType::kRelu1;
+  // Relu1 has a min raneg of -1, we need to quantize this
```

Comment:
"raneg" -> "range"

Reply:
@suharshs , thanks for the review, i have updated the code as per your comments, kindly check and approve.

Regards
Amit